### DISCUSSION TITLE
Use at least 1 donation platform that is NOT Paypal?

### DISCUSSION THREAD

**Discussion by Pentaphon** [transmission/transmission#3810] [Ideas]
I noticed the donation page on the homepage features only Paypal and I think a lot of people are wary of Paypal these days so it would benefit the project if it had at least 1 very open, very popular form of accepting donations that isn't Paypal. 

This is totally up to the dev team, but if you are one of those people that dislike Paypal, you should suggest an alternative that you do like and maybe the team will add it someday.

**Comment by Petrprogs** [comment]
Maybe [Buy me a coffee](buymeacoffee.com)? Or crypto

**Comment by mikeziri** [comment]
Came here to add an idea but saw this one.
I would like to donate but I don't have neither want to create a paypal account for that.
Are you able to setup the account to process credit card payments without an account?

Ideally I would prefer to donate via Lightning Network (BTC).

**Reply by Pentaphon** [reply]
> Came here to add an idea but saw this one. I would like to donate but I don't have neither want to create a paypal account for that. Are you able to setup the account to process credit card payments without an account?

It seems you can pay with a credit card but the Paypal payment page asks for an email as well. Not sure if that would deter you. This is why I think it's important to have at least 1 open-source friendly payment processor for people who don't want give any info to Paypal. 

> Ideally I would prefer to donate via Lightning Network (BTC).

>Or crypto

I'm not sure a simple donation page for an open source project needs crypto and the team might prefer not have to deal with the added complexity of crypto payments. I was thinking more like https://github.com/sponsors or https://opencollective.com/

**Comment by nixorokish** [comment]
+1 Please accept crypto donations. Would donate if there were crypto donation support

**Reply by mikalcallahan** [reply]
Just making some noise that I would like to donate _without_ a PayPal account - I don't mind if it's Github Sponsors, crypto, etc

**Comment by DMantis** [comment]
I would also like to donate with either ethical platforms (e.g. liberapay), direct card payment (e.g. stripe) or crypto.

In other words, in any modern way, that doesn't ask me to:
- create accounts;
- accept privacy-invasive TOS with "partners" data sharing;
- share my email and personal details.

I use Transmission almost everyday for several years and don't mind some monthly subscription to support the project.

**Comment by adamralph** [comment]
I would also like to donate, but I will not use PayPal.

It seems like low hanging fruit to switch on GitHub Sponsors.

**Comment by robd003** [comment]
+1 for enabling GitHub Sponsors
